.azure-pipelines/templates/build.yml

```diff
@@ -0,0 +1,91 @@
+parameters:
+- name: py_version
+  type: string
+  default: ''
+- name: image
+  type: string
+  default: ''
+- name: ccache
+  type: boolean
+  default: false
+- name: OSX_VERSION
+  type: string
+  default: ''
+- name: publish
```

Comment:
In some places you have a parameter named `deploy`, it is a bit confusing which does what. Can there be a better name?

Reply:
`artifact` or `artifact_name` ?

Reply:
Artifact of what?

Reply:
it's the name of the artifact that azure publishes

Reply:
But where is that name used? For the download links?

Reply:
What download links?

Reply:
See, e.g., https://dev.azure.com/scipp/scipp/_build/results?buildId=4447&view=results under "Build and Test". Is this not what you are referring to?

Reply:
No. This is what I am refering to: https://dev.azure.com/scipp/scipp/_build/results?buildId=4371&view=artifacts&pathAsName=false&type=publishedArtifacts

But the name is actually created by `conda-build`. When you conda build on linux, it makes a directory called `linux-64`. I get this directory in the `PathtoPublish` and then append the python version to create the `ArtifactName` here: https://github.com/scipp/scipp/blob/81495409c67f4d56146fb1b07a08c4eabed0ea17/.azure-pipelines/templates/build.yml#L86

I think the `publish` name originally came from the `PathtoPublish`.

Reply:
Ok, basically what I thought then (my link links to your link, I think). `azure_artifact_name`?

Reply:
But it is not really the full artifact name, it's just the root, or the name of the directory created by conda build.
The artifact name is made up of the `linux-64` (the one we are talking about now) and the python version to make up `linux-64-py3.7`.
Maybe I should just call it `conda_package_root`?

Reply:
I don't really care, as long as it is clearer what is does when using the template, without having to look at the template in detail.